alphalens/tears.py

```diff
@@ -138,9 +139,9 @@ def create_factor_tear_sheet(factor,
                                                                                1,
                                                                                std_err=std_quant_daily)
 
-    factor_autocorrelation = perf.factor_rank_autocorrelation(factor,
-                                                              time_rule='D')
-
+    factor_autocorrelation = pd.concat(
+        [perf.factor_rank_autocorrelation(factor, period=p) for p in periods], axis=1)
```

Comment:
I'm not sure we should add this to the default tearsheet, as the more plots we generate, the more daunting it will be to look at. Maybe we can add a flag `autocorr_for_all_periods`?